### PR TITLE
If statement to allow vpn access

### PR DESCRIPTION
**Pull request by AwkwardBen**
Adding an if clause for adding vpn access.

### CODE REVIEW

File: how-to-docs/examples/rds-temp.yaml [hunk 1]
```diff
@@ -52,10 +52,12 @@ Resources:
         FromPort: 5432
         ToPort: 5432
         SourceSecurityGroupId: {{ get_stack_output(cf_conn, env + '-infra', 'ComputeDefaultSG') }}
+{% if vpn_access %}
       - IpProtocol: tcp
```
Comment (Joseph-Irving) [inline how-to-docs/examples/rds-temp.yaml]: you should have vpn_acccess set to false by default top with some kind of explanation

File: how-to-docs/examples/rds-temp.yaml [hunk 2]
```diff
@@ -52,10 +52,12 @@ Resources:
         FromPort: 5432
         ToPort: 5432
         SourceSecurityGroupId: {{ get_stack_output(cf_conn, env + '-infra', 'ComputeDefaultSG') }}
+{% if vpn_access %}
```
Comment (jaykeshur) [inline how-to-docs/examples/rds-temp.yaml]: you need to set the property at the top of the file

File: how-to-docs/examples/rds-temp.yaml (hunk 1 above)
Comment (AwkwardBen) [inline how-to-docs/examples/rds-temp.yaml]: Yep just remembered to add this, new commit on the way

File: how-to-docs/examples/rds-temp.yaml (hunk 2 above)
Comment (AwkwardBen) [inline how-to-docs/examples/rds-temp.yaml]: Added the vpn_access line

**Pull request merged**